portfolio/src/main/webapp/api-script/script.js

```diff
@@ -0,0 +1,87 @@
+mapOptions = {
+  center: { lat: 0, lng: 0 },
+  zoom: 3,
+  scaleControl: false,
+  streetViewControl: false,
+  mapTypeControlOptions: {
+    mapTypeIds: ["moon"]
+  }
+};
+
+
+async function loadMap() {  
+  const response = await fetch("/landing-data");
+  const payload = await response.json();
+
+  
+  var map = new google.maps.Map(document.getElementById("map"), mapOptions);
+
+  var moonMapType = new google.maps.ImageMapType({
+    getTileUrl: function (coord, zoom) {
+      var normalizedCoord = getNormalizedCoord(coord, zoom);
+      if (!normalizedCoord) {
+        return null;
+      }
+      var bound = Math.pow(2, zoom);
+      return (
```

Comment:
I'd recommend building this string using JavaScript's string interpolation feature: https://dmitripavlutin.com/string-interpolation-in-javascript/